mautic/mautic
# Removed duplicated CSS code which destroyed the map on the lead detail

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed issues  |  /

## Description
@mqueme  noticed that the map was too small and it threw a JS error. It was caused by duplicated CSS style after merging the chart PR. 

## Steps to reproduce the bug (if applicable)
Open a lead with some IP lookup data or follow instructions in https://github.com/mautic/mautic/pull/1481. The map should be small, the point shouldn't be visible and the JS console will hold an error.

## Steps to test this PR
Apply this PR and refresh the lead detail page. The map should be full sized, the point(s) visible and JS error gone.
